[CF1] upn edit (#19814)
cloudflare/cloudflare-docs: src/content/docs/cloudflare-one/identity/idp-integration/entra-id.mdx

To [test](/cloudflare-one/identity/idp-integration/#test-idps-in-zero-trust) that your connection is working, select **Test**.

#### UPN and email

If your organization's UPNs do not match users' email addresses, you must add a custom claim for email. For example, if your organization's email format is `user@domain.com` but the UPN is `u908080@domain.com`, you must create an email claim if you are configuring email-based policies.

> By default, Cloudflare will look for an `email` claim in the JSON response. If an `email` claim does not exist, it will look for the unique claim name you created to represent email (for example, `email_identifier`). Last, if neither option exists, Cloudflare will then look for the UPN claim sent in the JSON response.

Author: ```suggestion
By default, Cloudflare will look for an `email` claim in the JSON response. If an `email` claim does not exist, it will look for the unique claim name you created and configured in the Cloudflare dashboard to represent email (for example, `email_identifier`). Last, if neither claim exists, Cloudflare will look for the UPN claim sent in the JSON response.
```

Author: @ james, can you check if these proposed edits are correct? I wanted to make it clearer

Reviewer: Its user defined claim name > email > upn

> To receive an email claim in a JSON response from Microsoft Entra, you must:

Author: ```suggestion
To receive an email claim in the `id_token` JSON response from Microsoft Entra, you must:
```

Author: @ james, can you check if these proposed edits are correct? I wanted to make it clearer

Reviewer: I'd remove `JSON response`. So it would just be To receive an email claim in the `id_token` from Microsoft Entra, you must:
